torch/_inductor/codegen/cpp.py

```diff
@@ -2586,17 +2584,20 @@ def run(kernel):
             return
 
         def select_tiling_indices():
+            rw_index = []
             all_index = []
             for node in nodes:
-                rw = dependencies.extract_read_writes(node._body, *node._sizes)
-                all_index += [dep.index for dep in itertools.chain(rw.reads, rw.writes)]
+                rw_index += [*node._body.reads, *node._body.writes]
+                all_index += list(node._body.indexing_exprs.values())
+
             contig_vars = set()
             contig_vars_list = []
             non_contig_stride_const = set()
             non_contig_stride_other = set()
-            for index in all_index:
+
+            for index in rw_index:
                 for var in index.free_symbols:
-                    if not re.search(r"^d\d+$", var.name):
+                    if not re.search(r"^z\d+$", var.name):
```

Comment:
Was this change fixing an issue?  Were we silently skipping everything before?

If not, we should handle both the d/z prefix. 

If so, can we add a test  (or make things an error if we get an unexpected prefix).

Reply:
```dependencies.extract_read_writes(node._body, *node._sizes)``` return a index_exp using d prefix(cretae new index exp uing new d prefix), but ```node._body.reads, *node._body.writes``` default using z prefix, so I change this path.